pkg/proxy/logger/logger.go

```diff
@@ -136,8 +137,11 @@ func (logTags) Addressing(i AddressingInfo) LogTag {
 	return func(lr *LogRecord) {
 		ipstr, port, err := net.SplitHostPort(i.SrcIPPort)
```

Comment:
Good call, thanks. I'd switch to [`ParseAddrPort`](https://pkg.go.dev/net/netip#ParseAddrPort) instead though. We want to avoid `panic`ing in the unlikely case of a malformed `i.SrcIPPort`.